cpp/src/parquet/stream_writer_test.cc

```diff
@@ -23,73 +23,61 @@
 #include <memory>
 #include <utility>
 
-#include "arrow/io/api.h"
 #include "parquet/exception.h"
 
 namespace parquet {
 namespace test {
 
+template <typename T>
+using optional = StreamWriter::optional<T>;
+
+using char4_array_type = std::array<char, 4>;
```

Comment:
this is almost as long as the original type do you really need an alias?

Reply:
Yes the alias is needed because of macros which do not work because of the comma in the type name:

.../arrow/cpp/src/parquet/stream_writer_test.cc:180:76: error: macro "ASSERT_THROW" passed 3 arguments, but takes just 2
  180 |   ASSERT_THROW(writer_ << optional<std::array<char, 4>>(), ParquetException);